ddprof-lib/src/main/cpp/flightRecorder.cpp

```diff
@@ -907,8 +907,12 @@ void Recording::writeCpool(Buffer* buf) {
     buf->put8(0);
     buf->put8(1);
     // constant pool count - bump each time a new pool is added
-    buf->put8(12);
-
+    char cpCount = 12;
+    Dictionary* stringLabelMap = Profiler::instance()->stringLabelMap();
+    if (stringLabelMap->empty()) { cpCount--; }
```

Comment:
Sorry for the nitpicks, but could you change the formatting to have new line after `{` ?
(also in other places)
```suggestion
    if (stringLabelMap->empty()) {
        cpCount--;
    }
```

Reply:
Yep, done!, should have done it like this in the first place :wink: